app/models/game.rb

```diff
@@ -36,6 +36,17 @@ def check?
     kings_in_check?('white') || kings_in_check?('black')
   end
 
+  def forfeit(current_user)
+    if current_user.id == white_player_id
+      white_player_id.update_attributes(white_player_id.losses: white_player_id.losses + 1)
```

Comment:
For some reason white_player is nil. It shouldn't be. Take a look at the association for white player, make sure its setup correctly.

Test it out in a console. This should work

```
game = Game.find 1
game.white_player # will not be nil
game.black_player #will not be nil
```